src/index.js

```diff
@@ -1,4 +1,5 @@
 import iziToast from "izitoast";
+import initializeRoamJSSidebarFeatures from "./sidebar";

```

Comment:
Made sure most of the new functionality was contained to its own file

Reply:
Cheers!